test/MiddlewareListenerTest.php

```diff
@@ -76,6 +84,7 @@ public function testSuccessfullyDispatchesMiddleware()
         $application = $event->getApplication();
 
         $application->getEventManager()->attach(MvcEvent::EVENT_DISPATCH_ERROR, function ($e) {
+            die(var_dump($e->getParam('exception')->getMessage()));
```

Comment:
Um, `die()`? Why not just a `var_dump()` or capturing that info to a variable to emit with the `fail()` below?